### PR TITLE
Render stage support (actually)

### PR DESCRIPTION
**Pull request by IMS212**
Mostly clean code, and I've confirmed all sky related render stages work. (Have not confirmed stages relating to outlines)

### CODE REVIEW

**Comment by IMS212** [comment]
Yes, this fixes SEUS PTGI HRR 3.

**Comment by IMS212** [comment]
Everything works except for outline and destroy, will look into this.

File: src/main/java/net/coderbot/iris/pipeline/WorldRenderingPhase.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package net.coderbot.iris.pipeline;
+
+// TODO: Implement this! Right now, this is just here as a stub so packs depending on the macros compile.
```
Comment (coderbot16) [inline src/main/java/net/coderbot/iris/pipeline/WorldRenderingPhase.java:3]: This TODO seems outdated...

**Review by coderbot16 — CHANGES_REQUESTED** [review]
Looks like this conflicts, and I have a few minor comments. Other than that this looks very nice and is a great improvement over the initial PR.

**Comment by IMS212** [comment]
I've yet to find out the cause of outline and destroy, it seems like we need to set those as the program gets used.

**Comment by coderbot16** [comment]
Oh right, anything using the BufferSource can't be done by just injecting around calls in LevelRenderer.

**Comment by IMS212** [comment]
Fixed.

**Review by coderbot16 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**